packages/app/src/components/VisualizationOptions/Options/LegendSet.js

```diff
@@ -18,12 +18,28 @@ import LegendDisplayStrategy from './LegendDisplayStrategy'
 import { sGetUiOptions } from '../../../reducers/ui'
 import { acSetUiOptions } from '../../../actions/ui'
 
+import { options } from '../../../modules/options'
+
 import {
     tabSection,
     tabSectionOption,
     tabSectionTitle,
 } from '../styles/VisualizationOptions.style.js'
 
+const optionName = 'legendSet'
+const defaultValue = options[optionName].defaultValue
+
+const query = {
+    legendSets: {
+        resource: 'legendSets.json',
```

Comment:
The .json extension should not be necessary

Reply:
Fixed